x/pool-incentives/client/cli/cli_test.go

```diff
@@ -137,6 +142,7 @@ func (s *IntegrationTestSuite) TestGetCmdParams() {
 	}
 }
 
+//TestGetCmdLockableDurations tests lockable duration CLI query commands
```

Comment:
Can we add spaces everywhere where applicable please to be aligned with Go conventions

```suggestion
// TestGetCmdLockableDurations tests lockable duration CLI query commands
```